doc/LSST_DESC_Notes/0001_twinkles_design/index.md

```diff
@@ -1,104 +1,86 @@
 ![](./_static/header.png)
 
-LSST DESC Notes Template and Author Guidelines
-==============================================
+# Twinkles: Science Goals, Survey Design and Development Methodology
```

Comment:
These design note edits look like mine, from the design-note branch... Some pollution seems to have occurred. It looks harmless though.